engines/director/lingo/lingo-object.cpp

```diff
@@ -724,7 +724,7 @@ void LM::m_respondsTo(int nargs) {
 // Window
 
 Common::String Window::asString() {
-	return "window \"" + getName() + "\"";
+	return "Window \"" + getName() + "\"";
```

Comment:
Why? This could break things. You cannot do it like this, the original printed it in lowerase